<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Time Taken</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="digitCount">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_6">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Navigator</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Attempted</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLCDNumber" name="lcdNumber_5">
     <property name="digitCount">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Unattempted</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Total Questions</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLCDNumber" name="lcdNumber_6">
     <property name="digitCount">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Marks</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="digitCount">
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Correct</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Incorrect</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLCDNumber" name="lcdNumber_4">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Submit and View Result</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="6">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
